ODIN_II/SRC/simulate_blif.cpp

```diff
@@ -1331,8 +1374,9 @@ nnode_t** get_children_of_nodepin(nnode_t* node, int* num_children, int output_p
  */
 static void initialize_pin(npin_t* pin) {
     // Initialise the driver pin if this pin is not the driver.
```

Comment:
isn't it just sufficient to always just assign it no matter what? if you are going to check, just set and forget

Reply:
Im not sure I understand

Reply:
if pin is not driver, make it driver == set pin as driver

Reply:
And if there are multiple drivers?

Reply:
this check is there to make sure its an input_pin to a node, if it is but its not initialized go initialize the parent (output_pin of a node) which will not be initialized since its not an input pin.
with that in mind, the best way to acheive that would be to check the pin dire

Reply:
> if pin is not driver, make it driver == set pin as driver

misread there, appologies

Reply:
good as it is :) sorry I thought this was in another bit of code